service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1840,6 +1840,9 @@ components:
         lastModified:
           type: string
           format: date-time
+        isDeleted:
+          type: boolean
+          nullable: false
```

Comment:
One additional API change to support listing deleted studies -- return this flag for each study returned. @tjennison-google @dolbeew